R/build-readme.R

```diff
@@ -19,18 +19,26 @@ build_rmd <- function(files, path = ".", output_options = list(), ..., quiet = T
   rlang::check_installed("rmarkdown")
   save_all()
 
+  paths <- files
+  abs_files <- is_absolute_path(files)
+  paths[!abs_files] <- path(pkg$path, files[!abs_files])
+
+  ok <- file_exists(paths)
+  if (any(!ok)) {
+    stop(
+      "Can't find file(s): ",
+      paste0(encodeString(files[!ok], quote = "'"), collapse = ", "), ".",
```

Comment:
This should use cli and/or rlang::abort I think.

Reply:
Are you happy to bump minimum version of cli (now: 2.4.0) up to 3.0.0 to get `cli::cli_abort()`?

Reply:
Ah, I guess this tells me the answer is YES: https://github.com/r-lib/devtools/issues/2364.

Reply:
That cleaned this up nicely!